serveradmin/serverdb/tests/test_ip_addr_type.py

```diff
@@ -0,0 +1,450 @@
+import logging
```

Comment:
While checking for something like [dataProviders](https://phpunit.readthedocs.io/en/9.5/writing-tests-for-phpunit.html#writing-tests-for-phpunit-data-providers-examples-datatest-php) in django/python I found this:
https://www.hacksoft.io/blog/improve-your-tests-django-fakes-and-factories .. maybe you want to read through the factories, could be a possibility to reduce the test code here. But nothing for the PR.

Reply:
It would be nice to have a header in this file just like all others.

Reply:
It is great to have tests now.  Maybe you can add the command to run the tests to the README.